packages/webapp/.env

```diff
@@ -15,6 +15,8 @@ NEXT_PUBLIC_TABLE_ROWS_MAX_FETCH_PER_SEC = "10"
 NEXT_PUBLIC_EDEN_CONTRACT_ACCOUNT = "test.edev"
 NEXT_PUBLIC_AA_FETCH_AFTER="1633883520000"
 NEXT_PUBLIC_TOKEN_CONTRACT = "eosio.token"
+NEXT_PUBLIC_TOKEN_SYMBOL = "WAX"
+NEXT_PUBLIC_TOKEN_PRECISION = "8"
```

Comment:
suggestion: this is redundant because we derive the symbol and the precision from the `NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT`.

Reply:
I agree it's redundant. But I don't like the idea of deriving this info from `NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT` now that we're using it for more than minimum donation amount.

I propose we do it the other way around: make `NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT` simply a number (`NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT ="10"`) and derive the asset from that using the `NEXT_PUBLIC_TOKEN_SYMBOL` and `NEXT_PUBLIC_TOKEN_PRECISION`.

That seems more explicit to me now. Thoughts?

Reply:
Yeah, I share your thoughts. I think being more explicit is better indeed. 

About deriving the correct asset from `NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT ="10"` might not be needed as well... This way we are explicitly forcing the configuration to be the correct asset and avoid undesired effects. 

We could check to make sure if the precision and asset from `NEXT_PUBLIC_APP_MINIMUM_DONATION_AMOUNT` is consistent with the `NEXT_PUBLIC_TOKEN_SYMBOL` and `NEXT_PUBLIC_TOKEN_PRECISION` though.